### PR TITLE
Simplify the relay tutorial docker setup

### PR DESCRIPTION
**Pull request by j-rivero**
# 🎉 New feature

Simplify the docker setup for the relay example by removing the build.sh and run.sh scripts that were adding helpers for development but were really not needed to run the important code.

## Summary
The two commits are self-explanatory:

 * 5e55b3f41556dce13797fd8068385ea693758a0d simplifies the Dockerfile in place and move it to Ubuntu Noble
 * 9318d585deb58881dd66552081b315cd15e59d50 removes all the jazz added to run docker and replace it on the instructions

## Test it
I've verified that it builds just fine. @caguero verified that it runs fine.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.
gazebo-release) (as needed): <LINK>

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Ubuntu 22.04
```
Comment (caguero) [inline docker/Dockerfile:1]: Nit: 24.04

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        sudo \
+        gnupg \
+        lsb-release \
+        wget \
+        tzdata \
+ && apt-get clean
+
+# Install gazebo dependencies
+RUN /bin/sh -c 'echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' \
+ && /bin/sh -c 'echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-prerelease `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-prerelease.list' \
```
Comment (caguero) [inline docker/Dockerfile:37]: I used this for testing but I think we should remove it.

File: tutorials/08_relay.md [hunk 3]
```diff
@@ -29,25 +29,17 @@ Download the [build.bash](https://github.com/gazebosim/gz-transport/raw/gz-trans
...
-wget https://github.com/gazebosim/gz-transport/raw/gz-transport14/docker/run.bash
-mkdir gz-transport && wget https://github.com/gazebosim/gz-transport/raw/gz-transport14/docker/gz-transport/Dockerfile -O gz-transport/Dockerfile
-chmod +x build.bash run.bash
+wget https://github.com/gazebosim/gz-transport/raw/gz-transport14/docker/gz-transport/Dockerfile -O Dockerfile
 ```
 
 Now, it's time to build the Docker image:
 ```
-./build.bash gz-transport
+docker build -t gz-transport-relay - < Dockerfile
```
Comment (caguero) [inline tutorials/08_relay.md:37]: `docker build -t gz-transport-relay - < gz-transport/Dockerfile`?

**Review by caguero — CHANGES_REQUESTED** [review]
Retarget to `gz-transport14`?

File: tutorials/08_relay.md [hunk 4]
```diff
@@ -29,25 +29,17 @@ Download the [build.bash](https://github.com/gazebosim/gz-transport/raw/gz-trans
...
 ```
 
 Run your Docker container:
 ```
-./run.bash gz-transport
-```
-
-Inside the docker instance, go to the `example` directory:
```
[the remark this answers is not in the document]
Comment (j-rivero) [reply tutorials/08_relay.md:48]: The final `WORKDIR` at the end of the Dockerfile should leave you exactly in that directory ready to run the example binary https://github.com/gazebosim/gz-transport/pull/530/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR64

File: docker/Dockerfile (hunk 1 above)
Comment (j-rivero) [reply docker/Dockerfile]: [921aa51](https://github.com/gazebosim/gz-transport/pull/530/commits/921aa519291591b5117176ad8a3f374fd3988d26)

**Review by caguero — APPROVED** [review]

**Pull request merged**